src/sentry/static/sentry/app/views/eventsV2/table.tsx

```diff
@@ -120,12 +68,11 @@ class Table extends React.PureComponent<Props, State> {
       return;
     }
 
-    this.readColumns();
-
     this.fetchData();
   }
 
   componentDidUpdate(prevProps) {
+    // todo(leedongwei): Ensure that checking by address is not a problem
     if (this.props.location !== prevProps.location) {
```

Comment:
This could be an eventView method that compares with the last view. That would let us only refetch when columns or condition related values change.